modules/svpc_host_project/main.tf

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
I am not sure about the purpose of this module?  IIUC we just need to make the addition in core and plumb it to the root module.

Reply:
So, my idea is to give two examples, but it does not mean I need two modules :) 
Going to fix.